src/gsi-ncdiag/gsi_ncdiag.py

```diff
@@ -188,38 +188,41 @@
     'surface_temperature': 'surface_temperature',
     'surface_roughness': 'surface_roughness_length',
     'surface_height': 'surface_geopotential_height',
-    'landmask': 'Land_Fraction',
+    'landmask': 'land_area_fraction',
     'air_temperature': 'air_temperature',
     'air_pressure': 'air_pressure',
     'atmosphere_pressure_coordinate': 'air_pressure',
     'atmosphere_pressure_coordinate_interface': 'air_pressure_levels',
     'air_pressure_levels': 'air_pressure_levels',
     'atmosphere_absorber_01': 'humidity_mixing_ratio',
-    'atmosphere_absorber_02': 'mass_concentration_of_carbon_dioxide_in_air',
-    'atmosphere_absorber_03': 'mass_concentration_of_ozone_in_air',
+    'atmosphere_absorber_02': 'mass_fraction_of_carbon_dioxide_in_air',
+    'atmosphere_absorber_03': 'mass_fraction_of_ozone_in_air',
     'atmosphere_mass_content_of_cloud_01': 'atmosphere_mass_content_of_cloud_liquid_water',
     'effective_radius_of_cloud_particle_01': 'effective_radius_of_cloud_liquid_water_particle',
     'atmosphere_mass_content_of_cloud_02': 'atmosphere_mass_content_of_cloud_ice',
     'effective_radius_of_cloud_particle_02': 'effective_radius_of_cloud_ice_particle',
-    'Water_Fraction': 'Water_Fraction',
-    'Land_Fraction': 'Land_Fraction',
-    'Ice_Fraction': 'Ice_Fraction',
-    'Snow_Fraction': 'Snow_Fraction',
-    'Water_Temperature': 'Water_Temperature',
-    'Land_Temperature': 'Land_Temperature',
-    'Ice_Temperature': 'Ice_Temperature',
-    'Snow_Temperature': 'Snow_Temperature',
-    'Vegetation_Fraction': 'Vegetation_Fraction',
-    'Sfc_Wind_Speed': 'Sfc_Wind_Speed',
-    'Sfc_Wind_Direction': 'Sfc_Wind_Direction',
-    'Lai': 'Lai',
-    'Soil_Moisture': 'Soil_Moisture',
-    'Soil_Temperature': 'Soil_Temperature',
-    'Land_Type_Index': 'Land_Type_Index',
-    'Vegetation_Type': 'Vegetation_Type',
-    'Soil_Type': 'Soil_Type',
-    'Snow_Depth': 'Snow_Depth',
+    'Water_Fraction': 'water_area_fraction',
```

Comment:
thank you for fixing names as well! we'll have to update ufo to use those.

Reply:
@CoryMartin-NOAA, as long as the Absorbers are updated, can we also update Clouds?
For example,
`atmosphere_mass_content_of_cloud_liquid_water` (total column quantity)
needs to be
`mass_content_of_cloud_liquid_water_in_atmosphere_layer` (layer quantity)

See 
https://github.com/JCSDA/ufo/issues/385
and
https://github.com/JCSDA/ufo/blob/420e2e1db46f9b1c2e656ee4436d990846711c46/src/ufo/ufo_variables_mod.F90#L39

Reply:
@jjguerrette  sure, is it just the 6 you have in the linked ufo_variables_mod that need changed in the ioda-converters?

Reply:
Yes @CoryMartin-NOAA, only those 6.  Both ufo_variables_mod and ufo_crtm_utils_mod will need changes after the geovals files are updated.  I'm happy to make those code changes when it's time.  I have never created a geovals file though.

Reply:
Perfect thanks, @jjguerrette . Once I've got the issues with the units in the combined conventional files as well as the subsetted ones, I'll regenerate the geovals files as well as the obs files with units and push them to the repositories. Concurrently, perhaps the easiest way is for you to make the code changes but don't create a pull request and we can merge the two (code changes and new geovals files) into one PR?

Reply:
The code changes for condensates are in UFO branch feature/CF_names.  Feel free to push other name changes there.

Reply:
Thank you Cory!